lib/generators/paper_trail/install_generator.rb

```diff
@@ -31,6 +32,13 @@ def create_migration_file
       end
     end
 
+    def create_initializer
+      create_file(
```

Comment:
Ok then that settles it. Explicit is the way to go here. Disregard my comment.

Sent from my iPhone

> On Aug 25, 2016, at 4:50 PM, Nathan <notifications@github.com> wrote:
> 
> In lib/generators/paper_trail/install_generator.rb:
> 
> > @@ -31,6 +32,13 @@ def create_migration_file
> >        end
> >      end
> >  
> > +    def create_initializer
> > +      create_file(
> Looks like the deprecation warning is important for people who previously were tracking associations solely by the presence of the version_associations table. See #785 for reference
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>